### PR TITLE
setup for initial Repos

### PR DESCRIPTION
**Pull request by stbischof**

### CODE REVIEW

**Comment by stbischof** [comment]
@maho7791 can you please approve? and then use the command 

`/otterdog merge`

File: .project [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (netomi) [inline .project]: can you please remove that file and maybe add a .gitignore ?

Comment (stbischof) [reply .project]: i really wanted to have this file to easier integrate this into eclips ide,

if it does not harm, please accept,

if really an issue, i can remove

Comment (netomi) [reply .project]: well thats user-specific, so when everybody starts to commit his local configuration files we have a cluttered repo. Actually I think I should add a general .gitignore file that excludes things like .project and .idea and the like.

Comment (stbischof) [reply .project]: i will remove

Comment (stbischof) [reply .project]: is removed

**Review by netomi — APPROVED** [review]

**Pull request merged**